### ISSUE TITLE
swift wolf rending gale error

### ISSUE THREAD

**Issue by robohavier2** [sinkillerj/ProjectE#2366]
ProjectE verision= ProjectE-PE1.0.1.jar
forge version=47.3.10
Steps to reproduce: craft swift wolfs rending gale fly for a while and then it stops making you fly
what I expected: continuous creative flight.
what happened:Not continuous flight
Note:It works when I take it out of the ring slot when it stops letting me fly but I don't know.

**Comment by robohavier2** [comment]
> delete

**Comment by pupnewfster** [comment]
Do you still have emc in your inventory to fuel it? Be it things like coal or glowstone, or emc stored in a Klein star.

**Comment by robohavier2** [comment]
> Do you still have emc in your inventory to fuel it? Be it things like coal or glowstone, or emc stored in a Klein star.

Yes, I do. It works for a bit then it stops, even if I have a klein star, and then when I unequip it works but I don't want to have to do that continuously.

**Comment by sinkillerj** [comment]
We will need logs of the issue in order to provide further support.
